apache/superset
# feat: View all fav charts and dashboards

### SUMMARY
Allows to show fav Charts and Dashboards from the Welcome page, by clicking the "Favorite" tab and "View all". It also removes the "Favorite" tab for "Saved Queries" as this functionality is not implemented yet. 

### TEST PLAN
1. Login in Superset
2. From the Welcome page click on "Favorite" for the "Dashboards" collapse and click "View all". 
3. OR From the Welcome page click on "Favorite" for the "Charts" collapse and click "View all". 
4. Check that only fav Charts/Dashboards are shown.
5. Change the filter on the top from "Favorite": "Yes" to "Favorite": "None".
6. Check that no favorites are shown.
7. Change the filter on the top from "Favorite": "No" to "Favorite": "Any".
8. Check that both fav and non-fav are shown
9. Go back to the Welcome page and check that no "Favorite" tab is shown for "Saved Queries"

### ADDITIONAL INFORMATION


- [x] Has associated issue: #11984
